daml-lf/language/daml-lf.bzl

```diff
@@ -88,8 +88,8 @@ def mangle_for_java(name):
     return name.replace(".", "_")
 
 def _to_major_minor(v):
-    (majorStr, minorStr) = v.partition(".")
-    (int(majorStr), int(minorStr))
+    (majorStr, _, minorStr) = v.partition(".")
+    return (int(majorStr), int(minorStr))
```

Comment:
:stuck_out_tongue_closed_eyes:

Reply:
I will absolutely never learn this.

Reply:
I got stuck for half a day once on a missing `return`. It was one hour trying to debug the issue and a few more repeatedly slapping myself after I solved it.